infra/src/main/java/com/redhat/service/smartevents/infra/models/bridges/BridgeDefinition.java

```diff
@@ -0,0 +1,44 @@
+package com.redhat.service.smartevents.infra.models.bridges;
+
+import java.util.Objects;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.redhat.service.smartevents.infra.models.gateways.Action;
+
+public class BridgeDefinition {
```

Comment:
POJO to store the Error Handler definition.

I used a container class in case we ever want to add more.